### PR TITLE
Add RoutePath.msg

### PR DESCRIPTION
**Pull request by qswawrq**

### CODE REVIEW

File: cav_msgs/msg/RoutePath.msg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# This contains lanelet IDs of the route and the shortest path in the route
+
+# Timestamp and frame_id of the map
+Header header
+
+# Route name
+string route_name
+
```
Comment (msmcconnell) [inline cav_msgs/msg/RoutePath.msg]: Can you add a route id field. Or do you feel different version of route generated due to a carma cloud update would get reflected in the name?

Comment (qswawrq) [reply cav_msgs/msg/RoutePath.msg]: Added. Those two should be separate field of a route msg.

File: cav_msgs/msg/Route.msg [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
 Header header
 
 # The route id
-string routeID
+string route_id
 
-# The route name for display to users
-string routeName
+# The version of current route
+int32 route_version
```
Comment (msmcconnell) [inline cav_msgs/msg/Route.msg]: Might as well be uint32

File: cav_msgs/msg/Route.msg [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
 
-# The validation status of this route
-# True if this route can be safely used in the CARMA platform
-bool valid
+# Route name
+string route_name
+
+# LaneLet IDs of the route shortest path
+int32[] shortest_path_lanelet_ids
```
Comment (msmcconnell) [inline cav_msgs/msg/Route.msg]: Lanelet Ids are of type int64_t
So use int64 here

**Comment by msmcconnell** [comment]
This PR relates to Issue usdot-fhwa-stol/CARMAPlatform#501

**Pull request merged**